edx_exams/apps/lti/views.py

```diff
@@ -69,8 +91,8 @@ def end_assessment(request, attempt_id):
     LTI End Assessment
     """
     # TODO: Here we'd do all the end of assessment things.
-    exam_attempt = ExamAttempt.objects.get(pk=attempt_id)
-    exam = exam_attempt.exam
+    attempt = ExamAttempt.objects.get(pk=attempt_id)
+    exam = attempt.exam
```

Comment:
I changed the variable names to be more consistent with the rest of the application, so I'm going to ignore the Codecov warning for now.

Reply:
can you just pragma: no-cover this whole function?

Reply:
I could, but I was a little concerned we'd forget to remove it. I'll add a comment to https://2u-internal.atlassian.net/browse/MST-1845 to remove it once we add tests for this view.